docs/changelog.rst

```diff
@@ -6,10 +6,11 @@ A log of changes by version and date.
 ======= ========== =====
 Version Date       Notes
 ======= ========== =====
+1.53.0  06/28/2024 Added FlashArray 2.34 client
 1.52.0  06/17/2024 Added FlashBlade 2.14 client
 1.51.0  05/28/2024 Added FlashArray 2.33 client
 1.50.0  04/23/2024 Added FlashArray 2.32 client
-1.49.0  03/23/2024 Added FlashArray 2.31 client
+1.49.0  03/07/2024 Added FlashArray 2.31 client
```

Comment:
Revert :-)

Reply:
We don't have this stuff in release branch, this is not the first time. Let's add it to avoid having to do this every release.

Reply:
Done.
I also noticed you already mentioned the proper fix in the python-sdk repo. Cool :)